### DISCUSSION TITLE
Multivariate regression

### DISCUSSION THREAD

**Discussion by raul-parada** [blue-yonder/tsfresh#983] [Q&A]
Hi,

Is there an example on how to predict multiple variables? From several inputs (5) predict two of them. Thanks

**Comment by nils-braun** [comment]
I assume you are looking for feature selection?
We do not have an example for this use-case, but you might be able to re-use the concepts describe in https://github.com/blue-yonder/tsfresh/blob/main/notebooks/04%20Multiclass%20Selection%20Example.ipynb: extract features on the full data and then select features first for the first target, then for the second and then combine the selected features.
